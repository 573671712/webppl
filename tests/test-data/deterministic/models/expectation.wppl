expectation(categoricalERP({ps: [.4, .5, .1], vs: [0, 1, 2]}), idF);
