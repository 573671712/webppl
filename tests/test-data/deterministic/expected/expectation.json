{
  "result": 0.7
}
